index.bs

```diff
@@ -14,9 +14,16 @@ Mailing List Archives: https://lists.w3.org/Archives/Public/public-immersive-web
 
 Editor: Piotr Bialecki 114482, Google http://google.com/, bialpio@google.com
 
+Warning: custom
+Custom Warning Title: Unstable API
+Custom Warning Text:
+  <b>Parts of the API represented in this document are unstable and may change at any time.</b>
+  <p>The relevant unstable sections are marked as such.</p>
```

Comment:
I would suggest to drop this. Anyone implementing the API will see which sections are unstable but having a top level warning that's so visible would give the wrong signal to the casual reader.

Reply:
Removed.